tutorials/independence.py

```diff
@@ -141,6 +141,33 @@
 #
 # ------------
 #
+# **D-variable Hilbert Schmidt Independence Criterion (Dhsic)** is an extension of Hsic,
+# and it tests for the joint independence of d-random variables. More details can be
+# found in :class:`hyppo.multivariate.dhsic`. Note that unlike Hsic or Dcorr, there is no
+# fast version of the test. It always uses the permutation method to compute its p-value.
+#
+# .. note::
+#
+#    :Pros: - Accurate, powerful independence test for more than two random variables
+#           - Much faster than constructing a joint independence test from multiple
+#             pairwise independence tests
+#    :Cons: - Not guaranteed to be consistent
```

Comment:
what do you mean? You should be clear when it is consistent. There is no point adding a test that is not consistent most of the time